api-scanner/src/main/java/net/corda/plugins/ScanApi.java

```diff
@@ -37,10 +37,12 @@
     private static final int FIELD_MASK = Modifier.fieldModifiers();
     private static final int VISIBILITY_MASK = Modifier.PUBLIC | Modifier.PROTECTED;
 
+    private static final String CORDA_INTERNAL = "net.corda.core.CordaInternal";
     private static final Set<String> ANNOTATION_BLACKLIST;
     static {
        Set<String> blacklist = new LinkedHashSet<>();
        blacklist.add("kotlin.jvm.JvmOverloads");
+       blacklist.add(CORDA_INTERNAL);
```

Comment:
What happens if someone adds this annotation to an existing public API member? That's a compatibility break, right? I think we need to be including this annotation in the output.

Reply:
Hmmm, isn't this a new annotation that can only be applied to "property getters/setters" and new annotations? AIUI, the semantics here are that `@CordaInternal` things are not included in the public API, c.f. the `ScanApi.writeMethods()` function where `@CordaInternal` methods are _explicitly_ hidden (by design). If someone were to annotate an existing API method as `@CordaInternal` then that method would also completely disappear from the output and so the "API Stability" build would break. So what's the problem?